plugins/woocommerce/client/admin/config/core.json

```diff
@@ -23,6 +23,7 @@
 		"product-virtual-downloadable": true,
 		"product-external-affiliate": true,
 		"product-grouped": true,
+		"product-linked": true,
```

Comment:
This cannot be enabled by default in `production`, so it must be `false` here. And set to `true` in `development` `/woocommerce/plugins/woocommerce/client/admin/config/development.json`

Reply:
😱

Reply:
addressed https://github.com/woocommerce/woocommerce/pull/43006/commits/d335f4fc0baa2adc1e1db530372cf0df0b805b8a